front/src/Settings/index.js

```diff
@@ -1,19 +1,40 @@
 import React from "react";
 import PropTypes from "prop-types";
 import Turtle from "../Turtle";
+import styled from "styled-components";
+import Dropdown from "../Dropdown";
 
 /**
  * Show the settings of all turtles.
  * Author: H.E. van der Laan
+ * Author: T.T.P. Franken
  *
  * props:
  * - turtles: an array of Turtles
  * - onTurtleEnableChange: function to call when the turtle enable button is pressed
+ * - onChange: a function which dictates what happens when a dropdown is changed.
```

Comment:
Incorrect, this callback is not called when a turtle dropdown is changed. I'd recommend renaming it as onSettingsChange

Reply:
aight